### ISSUE TITLE
Sony Xperia Z1 : upload very very slow

### ISSUE THREAD

**Issue by mgageo** [mapillary/mapillary_issues#959]
On  a Sony Xperia Z1 compact, the upload is very slow.
The size of the "Mapillary folder" : 0 byte.
In th status bar, I have a count of 6/551, and this count decrease by 1 every 3 minutes.
My profile : http://www.mapillary.com/profile/gotpo

On the same wifi point, with a Motorala MotoG, it's need only 10 minutes for the same number of images.

**Comment by jesolem** [comment]
@peterneubauer will take a look. If you can send logs, that would be great.

**Comment by mgageo** [comment]
The mail of the debug log was sent, the count is now 3/547

**Comment by mgageo** [comment]
It's not a problem of Mapillary but of the Sony.
I test the internet performance of the Sony with Ookla Speedtest : only 0.2 Mbits of upload.
with the Motorola the speed is 10 Mbits.

**Comment by TafThorne** [comment]
I thought I was seeing a problem very like this in my Sony z3c. I've left it going all night and am on 86 of 264 this morning. Wi-Fi should have been  enabled app night as it was on charge and a Speedtest shows 3Mb+ upload rate.

**Comment by TafThorne** [comment]
Seems that I cannot upload .log files. Here is a link to the one I took this morning https://www.dropbox.com/s/2vynw4qqb8nzxa7/mapillary.log?dl=0 Does not seem to show much going on overnight?

**Comment by peterneubauer** [comment]
@TafThorne seems you were not able to log in to Mapillary. Can you restart the app and check that e.g. your feed is updating and the upload is better?

**Comment by TafThorne** [comment]
I did so and that seemed to help the uploads.  I have nothing on the
opening feed still, but I assumed that was normal on the opening screen?
I also seemed to have 9 non-existance photos hanging around from an old
sequence.  These were possibly blank ones left after I deleted a few
through the app.  I got rid of those by clearing the app data from the
system > settings menu.

**Comment by ravilacoya** [comment]
I have the same problem. Extremely slow upload. It takes 4-5 hours to upload just 275 photos. And I plan to take thousands of photos. Is there any way to fix this, or a work around it in the mean time?

**Comment by peterneubauer** [comment]
Mmh this can be a network problem, and the xperia image files are also around 10mb big. Alternatively, copy the files to a computer and upload from there with upload.py from the github mapillary_tools project.

**Comment by ravilacoya** [comment]
Hi,

I tried reducing the resolution for my two last sequences and it uploaded much faster.

But it happened that in the middle of my last sequence, I went on getting photos for a couple of km, but they kept locked to the same place. What could have happened? Could this be related to having reduced the resolution? I guess this sequence should be deleted from Mapillary.

The failed sequence is this one: https://www.mapillary.com/map/s/3VhDmKfPeRE2jkxzCFFRIg

Cheers.
